client/state/ui/editor/actions.js

```diff
@@ -20,13 +20,33 @@ export const MODAL_VIEW_STATS = {
  * Returns an action object to be used in signalling that the editor should
  * begin to edit the post with the specified post ID, or `null` as a new post.
  *
+ * @param  {Number}  siteId   Site ID
+ * @param  {?Number} postId   Post ID
+ * @param  {String}  postType Post Type
+ * @return {Object}           Action object
+ */
+export function startEditingPost( siteId, postId, postType ) {
```

Comment:
Do you think we should default `postType = 'post'`? Fine to not, though "post" is something of a default in WordPress (see [`wp_insert_post`](https://developer.wordpress.org/reference/functions/wp_insert_post/), [`WP_Query`](https://codex.wordpress.org/Class_Reference/WP_Query#Type_Parameters) `post_type` arguments).